app.js

```diff
@@ -216,6 +216,7 @@ journey(app, {
   },
   timeoutDelay: 2000,
   apiUrl: `${config.api.url}/appeals`,
+  apiDratUrl: `${config.api.url}/drafts`,
```

Comment:
Minor point, but the `f` is missing in `draft`

Reply:
👍